### PR TITLE
Add sign that shows the document is an eInvoice

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

File: pretix_zugferd/invoice.py [hunk 1]
```diff
@@ -325,30 +327,47 @@ def _zugferd_generate_document(self, invoice):
...
-
-        if str(invoice.invoice_to_country) in settings.COUNTRIES_OVERRIDE:
+            self.__zugferd = False
+        elif str(invoice.invoice_to_country) in settings.COUNTRIES_OVERRIDE:
             # It's not possible to generate a valid ZUGFeRD invoice with the recipient in a non-standard country code (e.g. Kosovo)
-            return fname, ftype, content
+            self.__zugferd = False
 
         try:
             xml = self._zugferd_generate_document(invoice).serialize(
```
Comment (wiffbi) [inline pretix_zugferd/invoice.py:356]: Should this be somehow fenced inside an `if self.__zugferd:`? Why try to generate the xml when we won’t attach it to the PDF anyway?

**Review by wiffbi — COMMENTED** [review]
I am not sure what the changes in `generate` want to achieve. I assume removing the early `returns` and moving to `self.__zugferd` is a try to always return an invoice, even if the xml-generation fails? If so, you would need to remove the `raise e` in line 364 (see my comment). Or is the intention behind it to not generate an invoice at all if the xml-generation fails to save on processing power?

File: pretix_zugferd/invoice.py [hunk 2]
```diff
@@ -325,30 +327,47 @@ def _zugferd_generate_document(self, invoice):
...
                 schema="FACTUR-X_" + self.schema
             )
         except Exception as e:
+            self.__zugferd = False
             logger.exception(
                 "Could not generate ZUGFeRD data for invoice {}".format(invoice.number)
             )
             raise e
```
[the remark this answers is not in the document]
Comment (raphaelm) [reply pretix_zugferd/invoice.py:364]: Correct, the raise must be gone

**Review by wiffbi — APPROVED** [review]
Passt. Ich hab noch flake8 && isort laufen lassen.

**Pull request merged**